packages/cicd/github-actions/src/migration/abstract-migration.adapter.ts

```diff
@@ -99,7 +99,7 @@ export abstract class AbstractMigrationAdapter implements MigrationAdapter {
         output = this.mergeConsecutiveSections(output, formatterAdapter);
```

Comment:
this while class must use ReadableContent instead of Buffer, to avoid need of converting Buffer to ReadableContent